x/programs/cmd/simulator/cmd/config.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"github.com/ava-labs/hypersdk/x/programs/runtime"
+)
+
+func newConfig(step *Step, config *Config) (*runtime.Config, error) {
+	return runtime.NewConfigBuilder(step.MaxFee).
+		WithEnableTestingOnlyMode(true).
+		WithBulkMemory(true).
```

Comment:
Could you leave a comment on why this is required? I think we should be pedantic as possible about explaining these WASM runtime configs.